tool.html

```diff
@@ -60,7 +60,7 @@ <h3 class='open-sans-sans-serif-text'>Download MetaSUB data</h3>
 			<h3 class='align-text-center open-sans-sans-serif-text'>Entropy Analysis</h3>
 		</div>
 		<div class='vertical-separator-from-top-50' ng-show='myCtrl.showAnalysis'>
-			<div class='align-text-center italic-font-size-25' ng-if='!myCtrl.showEntropy'>Analyzing {{myCtrl.fileName}} ...</div>
+			<div class='align-text-center italic-font-size-25 FileColor' ng-if='!myCtrl.showEntropy'>Analyzing {{myCtrl.fileName}} ...</div>

```

Comment:
@MAZHARMIK - Let us replace this with - `orange-text` as the style for orange-text is already present in `tool.css`

Reply:
OK sure

Reply:
solved. Thanks for pointing it out. I am closing this PR now ...